backoffice/www/handlers/base.py

```diff
@@ -0,0 +1,20 @@
+import tornado.locale
+import tornado.web
+
+
+class BaseBOHandler(tornado.web.RequestHandler):
+  """A base class for handlers."""
+
+  COOKIE = 'id'
+
+  def get_template_path(self):
+    return 'backoffice/www/templates/'
+
+
```

Comment:
it seems like we have more blank lines here than needed (?)
3 here, 1 between the two methods below.